src/zenml/orchestrators/base_orchestrator.py

```diff
@@ -88,6 +88,7 @@ class BaseOrchestrator(StackComponent, ABC):
     """
 
     _active_deployment: Optional["PipelineDeploymentResponse"] = None
+    supports_scheduling: ClassVar[bool] = False
```

Comment:
This seems like something that should be a property on the `BaseOrchestratorFlavor` instead? It might be useful to get this information without needing all the dependencies for a specific orchestrator installed.

Reply:
To be more exact, I think this should be on the `BaseOrchestratorConfig` as a `@property` if possible. 

We have similar propertis like `is_remote`, `is_valid` in our base `StackComponentConfig` and `is_synchronous` in our `BaseOrchestratorConfig`.

Reply:
@bcdurak , thanks, I'll move it there, but what are the benefits of having it as a property and not a class variable there?

Reply:
Addressed